framework/db/Schema.php

```diff
@@ -313,7 +313,7 @@ public function refreshTableSchema($name)
      */
     public function createQueryBuilder()
     {
-        return new QueryBuilder($this->db);
+        return Yii::createObject(QueryBuilder::className(), [$this->db]);
```

Comment:
IIRC not using createObject was on purpose to make it faster. @samdark ?

Reply:
Maybe @rob006 remembers.

Reply:
I don't think that efficiency is a concern here. `QueryBuilder` is created only once per `Schema` instance. And AFAIK `ColumnSchemaBuilder` is used only in migration-like context, so it should be super rare anyway and any overhead in these scenarios should negligible.